strategies.py

```diff
@@ -86,7 +93,8 @@ def __init__(self, network, seconds_per_move=5, simulations_per_move=0,
         self.root = None
         self.result = 0
         self.result_string = None
-        self.resign_threshold = -abs(resign_threshold)
+        self.resign_threshold = resign_threshold or FLAGS.resign_threshold
```

Comment:
it seems lke the flag value should take priority over the passed parameter, no?

Reply:
fixed this by making it clearer that sometimes we pass in -1 to disable resign threshold. (in which case we override the flag)

Reply:
ok, LGTM :)